src/main/java/dev/jbang/cli/App.java

```diff
@@ -398,7 +398,7 @@ public static int setup(boolean withJava, boolean force, boolean chatty) {
 		}
 		if (Util.getShell() == Util.Shell.bash) {
 			if (changed) {
-				System.out.println("Please start a new Shell for changes to take effect");
+				System.err.println("Please start a new Shell for changes to take effect");
```

Comment:
What does that fix exactly ?

Reply:
I mistakenly believed that when running with a shell there would never be a need to execute code so I printed the text to stdout, but when running on Windows you actually need to run PowerShell anyway to be able to make changes to the Environment. So the text needs to be printed to stderr to avoid that the startup script tries to execute it.

Reply:
okey, but is the url generated actually right ?